spec/features/create_audio_object_spec.rb

```diff
@@ -0,0 +1,156 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Create a new Audio object via Pre-assembly', type: :feature do
+  audio_druid = '' # used for preassembly manifest files
+
+  let(:start_url) { 'https://argo-stage.stanford.edu/items/register' }
+  let(:preassembly_project_name) { "IntegrationTest-Audio-#{RandomWord.nouns.next}" }
+  let(:source_id_random_word) { "#{RandomWord.adjs.next}-#{RandomWord.nouns.next}" }
+  let(:audio_source_id) { "audio-integration-test:#{source_id_random_word}" }
+  let(:object_label_random_words) { "#{RandomWord.adjs.next} #{RandomWord.nouns.next}" }
+  let(:object_label) { "audio integration test #{object_label_random_words}" }
+  let(:local_manifest_location) { 'tmp/manifest.csv' }
+  let(:local_media_manifest_location) { 'tmp/media_manifest.csv' }
+  let(:preassembly_bundle_dir) { '/dor/staging/integration-tests/audio-test' }
+  let(:remote_manifest_location) { "preassembly@sul-preassembly-stage:#{preassembly_bundle_dir}" }
+  let(:remote_audio_files_location) { "#{remote_manifest_location}/audio-object" }
+  let(:audio_files) do
+    [
+      'audio_a_m4a.m4a',
+      'audio_a_mp3.mp3',
+      'audio_a_wav.wav',
+      'audio_img_1.jpg',
+      'audio_img_1.tif',
+      'audio_pdf.pdf'
+    ]
+  end
+  # rubocop:disable Layout/IndentationWidth
+  let(:preassembly_manifest_csv) do
+# no leading spaces
+<<~CSV
+druid,object
+#{audio_druid},#{audio_druid}
+CSV
+  end
+  let(:media_manifest_csv) do
+# no leading spaces
```

Comment:
Can you explain why?  Doesn't the squiggly heredoc strip leading spaces?

Reply:
I believe it ought to. This would be a nice, small change to make.

Reply:
sweet!  I breezed by this.